model_tests.rb

```diff
@@ -193,6 +193,14 @@ def test_centralheatpumpsystem_rb
     result = sim_test('centralheatpumpsystem.rb')
   end
 
+  def test_chiller_reformulated_rb
+    result = sim_test('chiller_reformulated.rb')
+  end
+
```

Comment:
SAme, missing the `# TODO: To be added in the next official release after: 3.1.0` so highlevel_tests.rb are failing.

Reply:
Done in 0035850b20634300d63fb7d4166064379521f441